README.md

```diff
@@ -415,6 +415,8 @@ This Four Keys project defines the key metrics as follows:
 
```

Comment:
In the intro of the # Dashboard section, include the same image of the dashboard (left this same comment in other file), and include a link to the new metrics file for more details on how the metrics are calculated.

"For details about how Four Keys calculates each metric in this dashboard, see [Four key metrics calculation]()."